### ISSUE TITLE
Add alternative text to a node

### ISSUE THREAD

**Issue by johannchopin** [mermaid-js/mermaid#3279]
Hey to all the team 👋 

Would be really nice to have a tooltip opening when hovering a node with the mouse, just like you can do with the `title` attribute in html:
```html
<p title="tooltip description">A text with a native tooltip description</p>
```

<p title="tooltip description">A text with a native tooltip description</p>

---- 

In a svg you can have the equivalent by using a `title` element:

```html
<svg xmlns="http://www.w3.org/2000/svg">
  <g>
    <title>Tooltip description</title>
    <rect x="0" y="0" width="400" height="100" fill="red"></rect>
    <text x="0" y="50" font-family="Verdana" font-size="15">A svg node with a native tooltip description</text>
  </g>
</svg>
```

[See the live example ](https://jsfiddle.net/qbcxgt82/12/)

So my proposition would be to add a syntax maybe like:

```
flowchart LR
    A["A simple node"]
    title A "An alternative text tooltip"
```

What do you think about this proposition? I'm willing to help so I can also try to open a PR if you aggree with this one 👍

**Comment by jpadfield** [comment]
I know it is not quite the same but have you tried : https://mermaid-js.github.io/mermaid/#/flowchart?id=interaction
I am still trying to work out the best css but it seems to work to allow alt text for a node.

**Comment by johannchopin** [comment]
Mmmh yes it is something in this direction however it's not pure html and require some css adjustment according to your environment. Having the native title make the final svg consistent and accessible without any extra css or javascript.

**Comment by jpadfield** [comment]
True, I have not sorted the CSS yet still not sure what is best but a working system does not seem to need too much ... work in progress :-)

**Comment by knsv** [comment]
I think this is a good idea, will ponder the syntax a bit but the proposed syntax makes sense. The pondering piece is to investgate if this could/should be applicable for other diagrams as well.

**Comment by johannchopin** [comment]
@knsv I made a list of elements where it could be useful to enable the syntax. Would be cool to have it all but I think it's a bit to optimistic. We should prioritize and see where we start.

- Flowchart
  - node
  - subgraph
- Sequence diagrams
  - node
  - actor/participant
  - note
- Class diagrams
  - class
- State diagrams
  - start?
  - end?
- Entity Relationship Diagrams
  - entity
- User Journey Diagram
  - section
  - actor
  - score
- Gantt diagrams
  - task
  - section
- Pie chart diagrams
  - datakey
- Gitgraph Diagrams
  - branch

**Comment by xav-ie** [comment]
I am recently using Mermaid inside of Notion and am running into the issue that I cannot add style tags. If we cannot do the title tag as @johannchopin suggests, could we pivot to using directives?  My main issue is that I cannot see the tooltip unless I add CSS, which I cannot do in Notion.
